content/config_rb_server.md

```diff
@@ -88,10 +88,6 @@ application:
     **data_collector** then increase this value. Default value: <span
     class="title-ref">100</span>.
 
-### opscode-expander
-
-{{% server_tuning_expander %}}
-
 ### opscode-solr4
```

Comment:
@PrajaktaPurohit Would you like reference to `opscode-solr4` and `opscode-solr` removed also?

Reply:
@mjingle Yes, remove all `opscode-solr` and `opscode-solr4`. ALSO add a ### Elasticsearch section with the options from the google doc.

Reply:
@mjingle Yes.